src/irmin/tree.ml

```diff
@@ -233,14 +258,14 @@ module Make (P : Backend.S) = struct
 
     let clear t = clear_info t.info
 
-    let of_v (v : v) =
+    let of_v ~env (v : v) =
```

Comment:
Couldn't this be an optional argument with `Env.empty ()` as a default value?

Reply:
I prefer as it is because if forces us to explicitly acknowledge which `env` is the right one in the rest of the code

Reply:
Fair enough :+1: